template/README.md

```diff
@@ -475,6 +476,92 @@ Please refer to these two threads for reference:
 
 Create React App will add decorator support when the specification advances to a stable stage.
 
+## Fetching AJAX Requests
+
+This project includes a [fetch polyfill](https://github.com/github/fetch), which makes it easier to make web requests.
+
+The global `fetch` function allows to easily makes AJAX requests. It takes in a URL as an input and returns a `Promise` that resolves to a `Response` object. You can find more information about `fetch` [here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
+
+
+__About Promises:__ This project also includes a [Promise polyfill](https://github.com/then/promise) which provides a full implementation of Promises/A+. A Promise represents the eventual result of an asynchronous operation, you can find more information about Promises [here](https://www.promisejs.org/) and [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
+
+You can make a GET request like this:
+```javascript
+import React, { Component } from 'react';
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { repos: [] };
+  }
+
+  componentDidMount() {
+    this.fetchRepos(this.props.user);
+  }
+
+  fetchRepos(user) {
+    fetch(`https://api.github.com/users/${user}/repos`)
+      .then((response) => response.json())
+      .then((repos) => this.setState({ repos }))
+      .catch((error) => console.log('Error', error));
```

Comment:
Yep, @insin mentioned this later in a comment too. We should handle both `!ok` and actual errors in the same way, that is, by showing the message.